src/components/TodoItem/TodoItem.tsx

```diff
@@ -0,0 +1,63 @@
+import cn from 'classnames';
+import { Todo } from '../../types/Todo';
+
+interface Props {
+  todo: Todo,
+  selectedTodo: Todo | null,
+  handleSelectTodo: (todo: Todo) => void
+}
+
+export const TodoItem: React.FC<Props> = (
+  {
+    todo,
+    selectedTodo,
+    handleSelectTodo,
+  },
+) => {
+  const isSelectedTodoId = selectedTodo === todo;
+
+  return (
+    <tr
+      data-cy="todo"
+      className={cn(
+        { 'has-background-info-light': isSelectedTodoId },
+      )}
+      key={todo.id}
+    >
+      <td className="is-vcentered">{todo.id}</td>
+      <td className="is-vcentered">
+        {todo.completed && (
+          <span className="icon" data-cy="iconCompleted">
+            <i className="fas fa-check" />
+          </span>
+        )}
+      </td>
+      <td className="is-vcentered is-expanded">
+
+        <p className={cn(
+          { 'has-text-danger': !todo.completed },
+          { 'has-text-success': todo.completed },
```

Comment:
one of those can serve as a default one. It will be overwritten if meets another condition

Reply:
> one of those can serve as a default one. It will be overwritten if meets another condition

I don't get how to do it, this option doesn't work correctly 

`        <p className={cn(
          'has-text-danger',
          { 'has-text-success': todo.completed },
        )}`